libdevice/crt_wrapper.cpp

```diff
@@ -34,6 +34,19 @@ int memcmp(const void *s1, const void *s2, size_t n) {
   return __devicelib_memcmp(s1, s2, n);
 }
 
+#if defined(__NVPTX__) || defined(__AMDGCN__)
+// For AMD/Cuda those symbols will be provided by libclc.
+DEVICE_EXTERNAL size_t __spirv_GlobalInvocationId_x();
+DEVICE_EXTERNAL size_t __spirv_GlobalInvocationId_y();
+DEVICE_EXTERNAL size_t __spirv_GlobalInvocationId_z();
+DEVICE_EXTERNAL size_t __spirv_LocalInvocationId_x();
+DEVICE_EXTERNAL size_t __spirv_LocalInvocationId_y();
+DEVICE_EXTERNAL size_t __spirv_LocalInvocationId_z();
+DEVICE_EXTERNAL size_t __spirv_GlobalSize_x();
+DEVICE_EXTERNAL size_t __spirv_GlobalSize_y();
+DEVICE_EXTERNAL size_t __spirv_GlobalSize_z();
+#endif // __NVPTX__ || __AMDGCN__
```

Comment:
Should this declarations be in the libdevice/spirv_vars.h header file? For the case if there will be other cpp files using those functions.